src/palace/manager/feed/annotator/base.py

```diff
@@ -243,6 +244,21 @@ def content(cls, work: Work | None) -> str:
 
 
 class Annotator(ToFeedEntry):
+    @classmethod
+    def _sample_link_comparator(cls, link1: Link, link2: Link):
+        link_1_type = link1.type.lower()
+        link_2_type = link2.type.lower()
+        if "/epub" in link_1_type and "/epub" not in link_2_type:
```

Comment:
Could we use the type constants, rather than strings, for this?

This might also obviate the need for `...type.lower()` and it's associated type checking errors.